assembly/data/Type_Tables.s

```diff
@@ -14,10 +14,10 @@
 
 @;format: attacking type, defending type, damage multiplier
 @;the multiplier is a (decimal) fixed-point number:
-@�2.0 TYPE_MUL_SUPER_EFFECTIVE
-@�1.0 TYPE_MUL_NORMAL
-@;�0.5 TYPE_MUL_NOT_EFFECTIVE
-@;�0.0 TYPE_MUL_NO_EFFECT
+@×2.0 TYPE_MUL_SUPER_EFFECTIVE
```

Comment:
corrected encoding of this file. Not everyone uses western (windows) and this file is treated as junk in my OS. pls make sure you encode text files with UTF-8.